templates/core/stories.html

```diff
@@ -0,0 +1,312 @@
+        <p>
+          <a href="/engage/future-proof-iot">Future-proofing Fingbox, the IoT home network monitoring device&nbsp;&rsaquo;</a>
+        </p>
+      </div>
+      <div class="col-4 u-vertically-center u-align--center u-hide--small">
+          {{ image (
+            url="https://assets.ubuntu.com/v1/258ac4a1-logo.png",
+            alt="Fing",
+            width="180",
+            height="73",
+            hi_def=True,
+            loading="auto|lazy"
+            ) | safe
+          }}
+      </div>
+    </div>
+    <div class="u-fixed-width">
+      <hr class="p-separator" />
+    </div>
+    <div class="row u-equal-height">
+      <div class="col-8">
+        <h4>How Domotz streamlined provisioning of IoT devices</h4>
+        <blockquote class="p-pull-quote--small">
+          <p class="p-pull-quote__quote">I absolutely believe that Ubuntu Core and snaps give us a competitive advantage. We are the only company in the IoT network management space that can guarantee a secure, always-up-to-date device for our customers’ on-premises deployments.</p>
+          <cite class="p-pull-quote__citation">Giancarlo Fanelli, CTO, Domotz</cite>
+        </blockquote>
+      </div>
+      <div class="col-4 u-vertically-center u-align--center u-hide--small">
+          {{ image (
+            url="https://assets.ubuntu.com/v1/84d82478-domotz.svg",
+            alt="domotz",
+            width="220",
+            height="48",
+            hi_def=True,
+            loading="lazy"
+            ) | safe
+          }}
+      </div>
+    </div>
+    <div class="row">
+      <p>
+        <a href="/engage/domotz-case-study">How Domotz streamlined provisioning of IoT devices&nbsp;&rsaquo;</a>
+      </p>
+      <p>
+        <a href="/internet-of-things/appstore">Learn more about Ubuntu Core in consumer applications&nbsp;&rsaquo;</a>
+      </p>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip--square-suru is-slanted--top-right">
+  <div class="u-fixed-width">
+    <h2>Get to market quickly with SMART START</h2>
+    <p class="u-sv3">
+      <a class="p-button--positive" href="/smartstart">Launch your smart product</a>
+    </p>
+  </div>
+</section>
+
+<section class="p-strip--light">
+  <div class="row p-divider">
+    <div class="col-4 p-divider__block">
+      <h4>Download Ubuntu Core</h4>
+      <p>Try Ubuntu Core 18 on one of a popular development board</p>
+      <p><a class="p-button--neutral p-link--external" href="https://cdimage.ubuntu.com/ubuntu-core/18/stable/current/">Download and install Ubuntu Core 18</a></p>
+    </div>
+    <div class="col-4 p-divider__block">
+      <h4>Make your own Ubuntu Core device</h4>
+      <p>Walk through the steps to build an image for a device.</p>
+      <p><a class="p-button--neutral p-link--external" href="https://docs.ubuntu.com/core/en/guides/build-device/image-building?from=corepage">Build a custom Ubuntu Core image</a></p>
```

Comment:
this link should be - https://ubuntu.com/core/docs/image-building - perhaps the copy doc is out of date

```suggestion
      <p><a class="p-button--neutral" href="/core/docs/image-building">Build a custom Ubuntu Core image</a></p>
```